Index: salsa/tensorflow/tensorflow.bzl
===================================================================
--- salsa.orig/tensorflow/tensorflow.bzl
+++ salsa/tensorflow/tensorflow.bzl
@@ -620,7 +620,7 @@ def tf_cc_shared_object(
             deps = deps,
             linkshared = 1,
             data = data + data_extra,
-            linkopts = linkopts + _rpath_linkopts(name_os_full) + select({
+            linkopts = linkopts + select({
                 clean_dep("//tensorflow:ios"): [
                     "-Wl,-install_name,@rpath/" + soname,
                 ],
